packages/fxa-settings/src/components/Settings/DropDownAvatarMenu/index.tsx

```diff
@@ -10,9 +10,10 @@ import { useEscKeydownEffect } from '../../../lib/hooks';
 import { ReactComponent as SignOut } from './sign-out.svg';
 import { logViewEvent, settingsViewName } from 'fxa-settings/src/lib/metrics';
 import { Localized, useLocalization } from '@fluent/react';
+import firefox from '../../../lib/firefox';
```

Comment:
Q: In the file above we used this:
```js
import firefox from 'fxa-settings/src/lib/firefox';
```

Is there a difference/benefit to using "../../../lib" vs "fxa-settings/src/lib"?
Not a big deal either way since it seems to be mixed above and in earlier files, but just curious if we should standardize on one.

Reply:
Many times VSCODE completes the import. It would be good to be consistent. Personally I like the relative paths. Not sure how others feel.